changelog.md

```diff
@@ -1,5 +1,21 @@
+## 4.1.0 (unreleased)
+
+* Separate Dragonfly from Images and Resources. [#3289](https://github.com/refinery/refinerycms/pull/3289). [Anita Graham](https://github.com/anitagraham)
+
+* [See full list](https://github.com/refinery/refinerycms/compare/4-0-stable...4.1.0)
+
+## 4.0.1 [unreleased]
```

Comment:
If you set this to `20 October 2017` then I'll release it today - also we need this PR for `4-0-stable` branch without the `4.1.0` stuff